ivadomed/utils.py

```diff
@@ -313,27 +325,30 @@ def run_eval(self):
         return dct
 
 
-def save_nii(data_lst, z_lst, fname_ref, fname_out, slice_axis, debug=False, unet_3D=False, binarize=True):
-    """Save the prediction as nii.
-        1. Reconstruct a 3D volume out of the slice-wise predictions.
-        2. Re-orient the 3D array accordingly to the ground-truth orientation.
+def pred_to_nii(data_lst, z_lst, fname_ref, fname_out, slice_axis, debug=False, kernel_dim='2d', bin_thr=0.5):
+    """Convert the NN predictions as nii.
 
-    Inputs:
-        data_lst: list of the slice-wise predictions.
-        z_lst: list of the slice indexes where the inference has been performed.
-              The remaining slices will be filled with zeros.
-        fname_ref: ground-truth fname
-        fname_out: output fname
-        slice_axis: orientation used to extract slices (i.e. axial, sagittal, coronal)
-        debug:
+    Based on the header of fname_ref image, it creates a nibabel object from the NN predictions (data_lst),
```

Comment:
ah! i see, it is a nibabel object. I would make it clear in the title of the function: " Convert the NN predictions as a nibabel object (nii)"